src/modules/Client/Controller/Admin.php

```diff
@@ -33,7 +33,6 @@ public function fetchNavigation()
                 'label' => __trans('Clients'),
                 'uri' => $this->di['url']->adminLink('client'),
                 'class' => 'contacts',
-                'sprite_class' => 'dark-sprite-icon sprite-users',
```

Comment:
These aren't referenced anywhere which I noticed while working on this module. I removed them to make things more clear